### PR TITLE
Added HTML examples for tables and lists

### PR DESCRIPTION
**Pull request by anenadic**
This PR addresses #210. The changes include:

- adding HTML table and list tags to the callout with all the HTML tags in the introduction episode (given that they are mainly used in the following examples throughout the rest of the lesson)
- slightly modified layout of the above callout - it now renders as a list to aid readability
- adding HTML table and list example snippets below the basic "Hello, World!" example (to draw people's attention to these two specific tags as we will use tables and lists in the following examples throughout the rest of the lesson)
- replacing the HTML table example with simple Markdown list in "Arrays"/"Loops and Collections" episode because it is simpler and won't add cognitive overload to explaining for loops
- a few more small typo fixes spotted throughout

### CODE REVIEW

File: _episodes/introduction.md [hunk 1]
```diff
@@ -77,11 +77,13 @@ a web browser will make a best guess regarding the layout of HTML elements on th
 > In the `Hello world` example above 5 different tags are used (`html`, `head`, `title`, `body` and `p`) in their open `<>` and closed `</>` form.
 > We see also the special `doctype` tag that indicates the format and version of the document, in this case, [HTML(5)][html5-wikipedia].
 >
-> Other examples include frequently used tags for structural elements such as `div`, `span`, `nav`, `section`;
-> stylized tags `i`/`em`, `b`/`strong` and `u` for *italics/emphasis*, **bold** and <u>underlined text</u>;
-> headings numbered from `h1` to `h6` for titles and progressively smaller sub-titles;
-> `img`, `video`, `audio` to embed rich media;
-> and the important `a` tag (anchor) used to [link](#) to sections in the same or other documents.
+> Other examples of frequently used tags are for:
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
> Many other tags exist to define: 
```

File: _episodes/introduction.md [hunk 2]
```diff
@@ -91,19 +93,52 @@ a web browser will make a best guess regarding the layout of HTML elements on th
...
+{: .challenge }      
+
+Let's write a more complex HTML example using a table showing the "Hello, World!" text in different languages that renders like: 
+![HTML table example](../fig/html-table.png){: width="600px"}
+
+The HTML to produce such a table looks like this (you can copy+paste the snippet into the HTML file you created in the previous example):
+~~~
+<table>
+    <tr><th>Language</th><th>Text</th></tr>
+    <tr><td>English </td><td>Hello, World!</td></tr>
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
    <tr><td>English</td><td>Hello, World!</td></tr>  
```

File: _episodes/introduction.md [hunk 3]
```diff
@@ -91,19 +93,52 @@ a web browser will make a best guess regarding the layout of HTML elements on th
...
+    <tr><th>Language</th><th>Text</th></tr>
+    <tr><td>English </td><td>Hello, World!</td></tr>  
+    <tr><td>French</td><td>Bonjour, le monde!</td></tr>
+    <tr><td>Portuguese</td><td>Olá, Mundo!</td></tr>
+    <tr><td>Serbian</td><td>Zdravo, svete!</td></tr>
+</table>
+~~~ 
+{: .language-html }
+
+Each row in enclosed between `<tr>` and `</tr>` tags; within a row, `<th>` and `</th>` tags are used to contain table headings
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
Each row in enclosed between **t**able **r**ow `<tr>` and `</tr>` tags; within a row, `<th>` and `</th>` tags are used to contain **t**able **h**eadings. 
```

File: _episodes/introduction.md [hunk 4]
```diff
@@ -91,19 +93,52 @@ a web browser will make a best guess regarding the layout of HTML elements on th
...
+    <tr><td>English </td><td>Hello, World!</td></tr>  
+    <tr><td>French</td><td>Bonjour, le monde!</td></tr>
+    <tr><td>Portuguese</td><td>Olá, Mundo!</td></tr>
+    <tr><td>Serbian</td><td>Zdravo, svete!</td></tr>
+</table>
+~~~ 
+{: .language-html }
+
+Each row in enclosed between `<tr>` and `</tr>` tags; within a row, `<th>` and `</th>` tags are used to contain table headings 
+(special table cells displayed in bold font) while regular table data cells are contained within `<td>` and `</td>` tags.
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
(special table cells displayed in bold font) while regular **t**able **d**ata cells are contained within `<td>` and `</td>` tags.
```

File: _episodes/introduction.md [hunk 5]
```diff
@@ -91,19 +93,52 @@ a web browser will make a best guess regarding the layout of HTML elements on th
...
+<ul>
+    <li>English: Hello, World!</li>  
+    <li>French: Bonjour, le monde!</li>
+    <li>Portuguese: Olá, Mundo!</li>
+    <li>Serbian: Zdravo, svete!</li>
+</ul>
+~~~ 
+{: .language-html }
+
+Here, we used unordered list tags `<ul>` and `</ul>` to define a list with 4 items, each in turn wrapped in
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
Here, we used **u**nordered **l**ist tags `<ul>` and `</ul>` to define a list with 4 items, each in turn wrapped in 
```

File: _episodes/introduction.md [hunk 6]
```diff
@@ -91,19 +93,52 @@ a web browser will make a best guess regarding the layout of HTML elements on th
...
+    <li>English: Hello, World!</li>  
+    <li>French: Bonjour, le monde!</li>
+    <li>Portuguese: Olá, Mundo!</li>
+    <li>Serbian: Zdravo, svete!</li>
+</ul>
+~~~ 
+{: .language-html }
+
+Here, we used unordered list tags `<ul>` and `</ul>` to define a list with 4 items, each in turn wrapped in 
+individual list item (`<li>` and `</li>`) tags.
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
individual **li**st item (`<li>` and `</li>`) tags. 
```

**Review by unode — APPROVED** [review]
This looks great!

Added a few minor suggestions and some bold for tag mnemonics.

**Comment by anenadic** [comment]
Nice one @unode, thank you for the review and improvements!

**Review by annefou — APPROVED** [review]
This is absolutely awesome! Thank you!

**Pull request merged**